spec/lib/alaveteli_text_masker_spec.rb

```diff
@@ -81,6 +81,15 @@
         expect(result).to eq(expected)
       end
 
+      it 'replaces UTF-8 double width characters' do
+        data = 'Héllø world'
+        rule = CensorRule.new(text: 'Héllø')
+        result = class_instance.send(
+          :apply_binary_masks, data, censor_rules: [rule]
+        )
```

Comment:
I think we should continue to test the public interface here:

```diff
diff --git a/spec/lib/alaveteli_text_masker_spec.rb b/spec/lib/alaveteli_text_masker_spec.rb
index 10beea328..19e53d0d9 100644
--- a/spec/lib/alaveteli_text_masker_spec.rb
+++ b/spec/lib/alaveteli_text_masker_spec.rb
@@ -84,8 +84,8 @@ describe AlaveteliTextMasker do
       it 'replaces UTF-8 double width characters' do
         data = 'Héllø world'
         rule = CensorRule.new(text: 'Héllø')
-        result = class_instance.send(
-          :apply_binary_masks, data, censor_rules: [rule]
+        result = class_instance.apply_masks(
+          data, 'application/vnd.ms-word', censor_rules: [rule]
         )
         expect(result).to eq 'xxxxxxx world'
       end
```

IDK if sending to `:apply_binary_masks` was a conscious choice though – happy to be convinced otherwise.